src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/jpa/Report.kt

```diff
@@ -69,7 +69,6 @@ class Report(
   /**
    * Human-readable reference. Previously known as ”incident number” in NOMIS.
    * A number when sourced from NOMIS.
```

Comment:
probs “A number when sourced from NOMIS _or DPS_.” instead? or can just be dropped?

Reply:
👍 

https://github.com/ministryofjustice/hmpps-incident-reporting-api/pull/164/commits/35d4dcea19de3d3c9de1a43a5b1eacf2de0c45be